boms/cloud-oss-bom/pom.xml

```diff
@@ -7,7 +7,7 @@
 
   <groupId>com.google.cloud</groupId>
   <artifactId>libraries-bom</artifactId>
-  <version>1.2.1-SNAPSHOT</version>
```

Comment:
Guava major version changed. It should be `2.0.0`

Reply:
good catch. done.